nuxt.config.ts

```diff
@@ -72,4 +72,11 @@ export default defineNuxtConfig({
       },
     ],
   },
+  typescript: {
```

Comment:
Since I couldn't find any justification - why was this option added? The documentation states that this is not recommended (https://nuxt.com/blog/v3-8#type-import-changes), and also the typescript documentation (https://www.typescriptlang.org/tsconfig#verbatimModuleSyntax) looks like it was a good addition to have this on.

Reply:
I originally didn't have this enabled, but the amount of changed files grew quite a bit when going through and fixing the imports. I disabled it to keep the PR as small as possible, but I can enable it again and fix the imports if that is better?

Reply:
The current amount of PRs this would get in conflict with is comparably small - almost none, if I see it correctly. For the matter of keeping PRs small, those changes are all related to one reason, which is updating while keeping good code settings, which takes higher priority for me here. Having this setting enabled improves how well tree-shaking will work, which should reduce the amount of code we load in production. So if not yet, when? 😎